### PR TITLE
Hints

### PR DESCRIPTION
**Pull request by kennethlove**
I think I'm ready to release.

### CODE REVIEW

**Review by kennethlove — COMMENTED** [review]
I dunno about you but I think it's effin' great

**Review by kennethlove — COMMENTED** [review]
I still like it.

File: src/brackets/exceptions.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+"""Exceptions for django-brackets mixins."""
+from django.core.exceptions import ImproperlyConfigured
+
+
+class BracketsConfigurationError(ImproperlyConfigured):
```
Comment (chrisjones-brack3t) [inline src/brackets/exceptions.py:5]: 👍🏻

File: src/brackets/mixins/access.py [hunk 2]
```diff
@@ -9,17 +9,17 @@
...
 from django.utils.timezone import now
 
+from brackets.exceptions import BracketsConfigurationError
+
 from .redirects import RedirectMixin
 
 if TYPE_CHECKING:  # pragma: no cover
     from collections.abc import Callable
-    from typing import Any, Optional, TypeAlias
+    from typing import Any
```
Comment (chrisjones-brack3t) [inline src/brackets/mixins/access.py:20]: Does this need to be here? I swore the `if TYPE_CHECKING:` was mainly for preventing circular imports?

File: src/brackets/mixins/http.py [hunk 3]
```diff
@@ -2,53 +2,63 @@
...
+if TYPE_CHECKING:  # pragma: no cover
     from collections.abc import Callable
 
-    from django.http import HttpRequest, HttpResponse
 
 __all__ = ["AllVerbsMixin", "HeaderMixin", "CacheControlMixin", "NeverCacheMixin"]
 
+A: TypeAlias = tuple[Any]
+K: TypeAlias = dict[str, Any]
+_View: TypeAlias = tuple[HttpRequest, tuple[Any], dict[str, Any]]
```
Comment (chrisjones-brack3t) [inline src/brackets/mixins/http.py]: Eh. I mean I get it but it's kind of ugly.

**Review by chrisjones-brack3t — APPROVED** [review]
This was a ton of code. I skimmed a good amount. I didn't see anything off. My only real question was the `if type checking` stuff.

**Pull request merged**